openmdao/core/group.py

```diff
@@ -1843,27 +1845,20 @@ def _setup_var_data(self):
 
         self._var_allprocs_abs2meta = allprocs_abs2meta
 
-        for prom_name, abs_list in allprocs_prom2abs_list['output'].items():
-            if len(abs_list) > 1:
-                self._collect_error("{}: Output name '{}' refers to "
-                                    "multiple outputs: {}.".format(self.msginfo, prom_name,
-                                                                   sorted(abs_list)))
-
-        for io in ('input', 'output'):
-            a2p = self._var_allprocs_abs2prom[io]
-            for prom, abslist in self._var_allprocs_prom2abs_list[io].items():
-                for abs_name in abslist:
-                    a2p[abs_name] = prom
+        # for prom_name, abs_list in allprocs_prom2abs_list['output'].items():
+        #     if len(abs_list) > 1:
```

Comment:
This check is commented out, but what does it mean? multiple outputs with the same path means two outputs promoted to the same name, but this message doesn't look like the one we get for that.

Reply:
The check was just moved over into NameResolver.  There's a test for the error message.  Do you recall what the different message looked like?

Reply:
I don't remember, but I feel like it is a clearer message than this.